wiki/Community/osu!_community_meetings/fr.md

```diff
@@ -1,3 +1,8 @@
+---
+outdated: true
+outdated_since: c5ee74153aa8ac008034cce4efe4be4350f8034b
+---
+
```

Comment:
```suggestion
```

Is it possible to update the article instead of outdate it? Thanks in advance

From L34 to L43:

``| Meeting n° | Lien | Résumé | Agenda |
| :-: | :-- | :-- |
| 0 | [19 septembre 2021](https://youtu.be/-LjNll8VYTM) | [Document de synthèse](https://gist.github.com/peppy/08b38cbac56a8b1e88133853a6a8d2fb) | Introduction à ces meetings, critères de classement, points de difficulté et de performance, osu!lazer en général |
| 1 | [3 octobre 2021](https://youtu.be/D3OZCm6szVE) | [Document de synthèse](https://docs.google.com/document/d/1-miPFZySYlYbtPW4G0828taycO1gh7dZ4icfSJCr4uA) | Questions diverses de la communauté, mise à jour concernant le développement |
| 2 | [17 octobre 2021](https://youtu.be/nLT-fLeogrw) | [Document de synthèse](https://docs.google.com/document/d/1wEQjzku_4IzvcWRanu0GkvWiPvb-IB7WzwU7Eyojj2E) | Questions diverses de la communauté |
| 3 | [31 octobre 2021](https://youtu.be/omvZbC7zJUI) | [Document de synthèse](https://docs.google.com/document/d/1bhaMx--chdys8AbzSDOItbv710SVhsww2BYPzQg2rvw) | Questions diverses de la communauté |
| 4 | [14 novembre 2021](https://youtu.be/rEWgTY-L95Y) | [Document de synthèse](https://docs.google.com/document/d/1rhrz0zaxQfVK1UnvrcEqj4UMagTD0bPNh8wby8I38IA) | Processus de développement des points de performance, diverses questions de la communauté, mise à jour concernant le développement |
| 5 | [28 novembre 2021](https://youtu.be/SqMrY26Q6Tw) | [Document de synthèse](https://docs.google.com/document/d/1R3OZZazb4p_PRcjEMvIrsfg5UqMyra-4QJd5qlvBb98) |  Questions diverses de la communauté, mise à jour concernant le développement |
| 6 | [11 décembre 2021](https://youtu.be/dtGjIyTFtMA) | [Document de synthèse](https://docs.google.com/document/d/1WtvmytxdzVTZX86JIWVgl4WZMTz3AfrMHOSVImEIMz8) | L'avenir des osu! World Cups, questions diverses sur la communauté |
| 7 | [18 décembre 2021](https://www.twitch.tv/videos/1236856715) | [Document de synthèse](https://docs.google.com/document/d/1SqCsuU210c5g7DF9iQ4POunV-U-k5CTBUCaDClfrSrA) | peppy répond à diverses controverses et drames dans la communauté |``

Reply:
thank you but try to format your suggestion to be a bit more readable (or just list the translations of the new items in the new columns in this case) next time

Reply:
also this is subject to change if peppy or other people have opinions on it, so a separate pr is better in this case.

Reply:
It was impossible for me to suggest directly in the places that interested me, as these were not modified, otherwise of course I would have done it.

Reply:
the problem is that i have to reformat and check it to make sure that nothing else changed (it's on one line and you missed an additional `:--` for the table alignment row) instead of just copy-pasting clearly readable items